bedrock/settings/base.py

```diff
@@ -952,7 +955,7 @@ def lazy_email_backend():
 }
 
 MOFO_SECURITY_ADVISORIES_PATH = config('MOFO_SECURITY_ADVISORIES_PATH',
-                                       default=path('..', 'mofo_security_advisories'))
+                                       default=path('mofo_security_advisories'))
```

Comment:
Ideally it would make more sense as a separate commit, but I'm comfortable merging as-is and will avoid further pedantry :)